docs/pipelines/process/stages.md

```diff
@@ -246,6 +246,11 @@ You control the dependencies by setting the triggers on each stage of the releas
 
 You can specify the conditions under which each stage runs. By default, a stage runs if it does not depend on any other stage, or if all of the stages that it depends on have completed and succeeded. You can customize this behavior by forcing a stage to run even if a previous stage fails or by specifying a custom condition.
 
+> [!NOTE]
+>
+> Condtions for failed('JOBNAME/STAGENAME') and succeeded ('JOBNAME/STAGENAME') as in the following example will only work for [YAML pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/stages?view=azure-devops&tabs=yaml).
```

Comment:
Can you remove /en-us/ from the URL string? It will cause problems for translation. 
```suggestion
> Condtions for failed('JOBNAME/STAGENAME') and succeeded ('JOBNAME/STAGENAME') as in the following example will only work for [YAML pipelines](https://docs.microsoft.com/azure/devops/pipelines/process/stages?view=azure-devops&tabs=yaml).
```
Can you remove /en-us/ from the URL string? It will cause problems for translation.